cmd/restic/cmd_backup.go

```diff
@@ -91,7 +91,7 @@ type BackupOptions struct {
 	ExcludeLargerThan       string
 	Stdin                   bool
 	StdinFilename           string
-	Tags                    []string
+	Tags                    restic.TagList
 	Host                    string
```

Comment:
Because I'm using `restic.TagList` here

Reply:
Oh, you're right it should be `restic.TagList`.